SPM Example/PortalSwift/ViewController.swift

```diff
@@ -1030,7 +1030,12 @@ class ViewController: UIViewController, UITextFieldDelegate {
           let availableRecoveryMethods = try await self.portal?.availableRecoveryMethods() ?? []
           let walletExists = try await self.portal?.doesWalletExist() ?? false
           let solanaWalletExists = try await self.portal?.doesWalletExist("solana") ?? false
-          let isWalletOnDevice = try await self.portal?.isWalletOnDevice() ?? false
+          var isWalletOnDevice = false
```

Comment:
why is this a breaking change? hasnt the isWalletOnDevice function always specified that it throws?